domains/priyanshusaini.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+      "username": "priyanshusaini105",
+      "email": "priyanshusaininew@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://priyanshusaini.co"

```

CNAME can't contain protocol
```suggestion
      "CNAME": "priyanshusaini.co"
```